lib/jekyll-mentions.rb

```diff
@@ -2,28 +2,36 @@
 require 'html/pipeline'
 
 module Jekyll
-  class Mentions < Jekyll::Generator
+  class Mentions < Jekyll::Converter
+    MENTIONABLE_EXTS = %w{.html .htm .md .markdown .mkdn .mkd .textile}
```

Comment:
So the mention filter doesn't know what it is allowed to filter? Hm. Could that functionality be ported to html pipeline? It's any language that becomes, eventually, HTML. Or maybe what we do is start passing in an optional output_ext in matches as well via Jekyll?